### PR TITLE
feat(link-group): add html, css, tokens

### PR DESCRIPTION
**Pull request by robbieatacato**
Solve: https://github.com/nl-design-system/denhaag/issues/837

Purpose
Adding Link Group:
- html
- css
- tokens

closes #839

### CODE REVIEW

File: components/LinkGroup/src/stories/bem.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-link-group">
+      <h3 class="utrecht-heading-4 denhaag-link-group__caption">Caption</h3>
+      <nav class="denhaag-link-group__navigation" aria-labelledby="denhaag-link-group-navigation-1">
+        <ul
+          id="denhaag-link-group-navigation-1"
```
Comment (paulAcato) [inline components/LinkGroup/src/stories/bem.stories.mdx:35]: Not sure if this needs an ID?

File: packages/components-css/src/index.tsx [hunk 2]
```diff
@@ -1,12 +1,10 @@
 import '../../../components/Alert/src/index.scss';
 import '../../../components/Accordion/src/index.scss';
 import '../../../components/AnchorNavigation/src/index.scss';
-import '../../../components/AuthenticationBlock/src/index.scss';
```
Comment (paulAcato) [inline packages/components-css/src/index.tsx:4]: Why are you removing blocks which area already on the `main` branch?

File: packages/components-css/src/index.tsx [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
 import '../../../components/BadgeCounter/src/index.scss';
 import '../../../components/Blockquote/src/index.scss';
 import '../../../components/Breadcrumb/src/index.scss';
 import '../../../components/Button/src/index.scss';
-import '../../../components/ButtonGroup/src/index.scss';
```
Comment (paulAcato) [inline packages/components-css/src/index.tsx:9]: Why are you removing blocks which area already on the `main` branch?

**Review by paulAcato — CHANGES_REQUESTED** [review]

File: components/LinkGroup/src/stories/bem.stories.mdx (hunk 1 above)
Comment (robbieatacato) [reply components/LinkGroup/src/stories/bem.stories.mdx:35]: @paulAcato Its for screen readers, see also MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#labeling_section_content

File: packages/components-css/src/index.tsx (hunk 2 above)
Comment (robbieatacato) [reply packages/components-css/src/index.tsx]: fixt it

File: components/LinkGroup/src/stories/bem.stories.mdx (hunk 1 above)
Comment (paulAcato) [reply components/LinkGroup/src/stories/bem.stories.mdx:35]: @robbieatacato check, looked like a bit of overkill to me ;-)

**Review by paulAcato — APPROVED** [review]

File: components/LinkGroup/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@gemeente-denhaag/link-group",
+  "description": "The Link Group component. List of links.",
+  "author": "Municipality of The Hague",
+  "license": "EUPL-1.2",
+  "version": "0.1.1-alpha.46",
```
Comment (bddjong) [inline components/LinkGroup/package.json:6]: We shouldn't start versioning at 46 for new components.

File: proprietary/Components/src/denhaag/link-group.tokens.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    "link-group": {
+      "image": {
+        "padding-block-end": { "value": "1rem" }
+      },
+      "caption": {
+        "color": { "value": "{denhaag.color.grey.5}" },
+        "font-weight": { "value": "500" }
+      },
+      "navigation-link": {
+        "line-height": { "value": "1.6875rem" }
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/link-group.tokens.json:12]: If the link is 18px, and line height is 27px, we should use `"value": "1.5"` in my opinion.

File: components/LinkGroup/README.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
+# Link Group
```
Comment (bddjong) [inline components/LinkGroup/README.md:1]: This documentation is titled link group, but a quick read tells me it's only talking about the Link itself, and nothing about link groups.

**Review by bddjong — COMMENTED** [review]
Still have some comments. I don't think there's anything we can do about the version number now though, since it's already merged.

**Pull request merged**